src/lib/PhpCsFixer/IbexaInternalConfigFactory.php

```diff
@@ -15,27 +15,52 @@
  *
  * @internal
  */
-class EzPlatformInternalConfigFactory
+final class IbexaInternalConfigFactory
 {
-    public const EZPLATFORM_PHP_HEADER = <<<'EOF'
+    public const IBEXA_PHP_HEADER = <<<'EOF'
```

Comment:
FYI: this change is an BC break by the book :stuck_out_tongue:

Reply:
You're right. Fixed in cf2c8af6.